src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.js

```diff
@@ -147,10 +147,14 @@ const VoltageLevelCreationDialog = ({
                 [LOW_VOLTAGE_LIMIT]: voltageLevel[LOW_VOLTAGE_LIMIT],
                 [HIGH_VOLTAGE_LIMIT]: voltageLevel[HIGH_VOLTAGE_LIMIT],
                 [LOW_SHORT_CIRCUIT_CURRENT_LIMIT]: unitToKiloUnit(
-                    voltageLevel.ipMin
+                    fromCopy
+                        ? voltageLevel?.identifiableShortCircuit?.ipMin
+                        : voltageLevel.ipMin
```

Comment:
?
without extension

Reply:
Yes, It depends on the formCopy if it's true we are getting infos from network-map-server else it's from network-modification ( the data model in network did not change)